frontend/src/routes/(authed)/player/[id]/+page.server.ts

```diff
@@ -42,7 +42,8 @@ export const load: PageServerLoad = async ({
   const lost = totalMatches - wins;
   const winrate = totalMatches > 0 ? wins / totalMatches : 0;
 
-  const mmr = mmrHistory[mmrHistory.length - 1]?.mmr ?? null;
+  const mmr =
+    totalMatches > 10 ? mmrHistory[mmrHistory.length - 1]?.mmr ?? null : null;
```

Comment:
Do you need this after adding the filter in backend?

Reply:
No, I'll refactor this.

Reply:
Fixed